Application/EdFi.Ods.Tests/EdFi.Ods.Api/Middleware/OdsInstanceSelectorTests.cs

```diff
@@ -29,7 +31,8 @@ public void SetUp()
         {
             _apiClientContextProvider = A.Fake<IApiClientContextProvider>();
             _odsInstanceConfigurationProvider = A.Fake<IOdsInstanceConfigurationProvider>();
-            _odsInstanceSelector = new OdsInstanceSelector(_apiClientContextProvider, _odsInstanceConfigurationProvider);
+            _odsRouteRootTemplateProvider = A.Fake<IOdsRouteRootTemplateProvider>();
+            _odsInstanceSelector = new OdsInstanceSelector(_apiClientContextProvider, _odsInstanceConfigurationProvider, _odsRouteRootTemplateProvider);
```

Comment:
More than just modifying this to get it to compile, it would be good to actually add some unit test coverage for this previously unidentified scenario.